src/System.Text.RegularExpressions/tests/project.json

```diff
@@ -1,27 +1,50 @@
 {
-  "dependencies": {
-    "Microsoft.NETCore.Platforms": "4.3.0-beta-devapi-24512-01",
-    "System.Collections": "4.3.0-beta-devapi-24512-01",
-    "System.Console": "4.3.0-beta-devapi-24512-01",
-    "System.Globalization": "4.3.0-beta-devapi-24512-01",
-    "System.Linq.Expressions": "4.3.0-beta-devapi-24512-01",
-    "System.ObjectModel": "4.3.0-beta-devapi-24512-01",
-    "System.Runtime": "4.3.0-beta-devapi-24512-01",
-    "System.Runtime.Extensions": "4.3.0-beta-devapi-24512-01",
-    "System.Text.RegularExpressions": "4.3.0-beta-devapi-24512-01",
-    "System.Threading.Tasks": "4.3.0-beta-devapi-24512-01",
-    "test-runtime": {
-      "target": "project",
-      "exclude": "compile"
+    "dependencies": {
+        "Microsoft.NETCore.Platforms": "4.3.0-beta-devapi-24512-01",
+        "System.Collections": "4.3.0-beta-devapi-24512-01",
+        "System.Collections.NonGeneric": "4.3.0-beta-devapi-24512-01",
+        "System.Console": "4.3.0-beta-devapi-24512-01",
+        "System.Globalization": "4.3.0-beta-devapi-24512-01",
+        "System.Linq.Expressions": "4.3.0-beta-devapi-24512-01",
+        "System.ObjectModel": "4.3.0-beta-devapi-24512-01",
+        "System.Runtime": "4.3.0-beta-devapi-24512-01",
+        "System.Runtime.Extensions": "4.3.0-beta-devapi-24512-01",
+        "System.Text.RegularExpressions": "4.3.0-beta-devapi-24512-01",
```

Comment:
Why add System.Text.RegularExpression reference here, when test csproj is referencing the pkgproj?